__tests__/main.test.js

```diff
@@ -135,4 +145,48 @@ describe('Articles', () => {
 			.send({ inc_votes: 'not-number' })
 			.expect(400)
 	})
+	it('POST 201 /api/articles/article_id/comments responds with added comment when posted', () => {
+		const body = {
+			body: 'We can do comments while testing do not we?',
+			author: 'icellusedkars',
+		}
+		return request(app)
+			.post('/api/articles/1/comments')
+			.set('Accept', 'application/json')
+			.send(body)
+			.expect(201)
+			.then(({ body: { comment } }) => {
+				expect(comment.body).toBe(body.body)
+				expect(comment.author).toBe(body.author)
+				expect(comment.article_id).toBe(1)
+			})
+	})
+	it('POST 400 /api/articles/article_id/comments responds with bad request when poseted non-exist author name', () => {
```

Comment:
I get what you mean because the user is unauthorised. However, technically this is a non-existent user not an invalid one, so you could send back a 404 with a 'user not found' message.